### DISCUSSION TITLE
Dev Stuff???? (PART 2)

### DISCUSSION THREAD

**Discussion by ThatLegend** [Prodigy-Hacking/ProdigyMathGameHacking#1179] [(A) Anything]
@ArcerionDev @PatheticMustan 
Hi, My Discord Is banned I have talked to them and they wont unban me (they think I got hacked).

**Comment by ArcerionDev** [comment]
can I have your tag?

**Comment by ThatLegend** [comment]
my dizzy is ban rip (YES DISCORD IP BANNED ME)
